lib/voxpupuli/acceptance/rake.rb

```diff
@@ -1,5 +1,4 @@
 begin
-  require 'puppetlabs_spec_helper/tasks/beaker'
   # Fixtures can be needed because of spec_prep
   require 'puppetlabs_spec_helper/tasks/fixtures'
 rescue LoadError
```

Comment:
I'm wondering if that's actuall a good idea. probably not. we should try to load puppetlabs_spec_helper/tasks/fixtures but always load beaker-rspec/rake_task. that's a runtime dep.

Reply:
Oh right. At the very least it should split the two so it can reliably fall back to beaker-rspec. The old puppetlabs_spec_helper did provide some more features, but we don't really use it. So I'd be OK with hard requiring beaker-rspec. If those features are needed, they can be ported to beaker-rspec.